releasenotes/notes/read-secrets-from-multiple-backends-a2cc225cd86c9256.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Added new executable `readsecret_multiple_backends.sh` that allows the agent
+    to read secrets both from files and Kubernetes secrets.
```

Comment:
Your comment implies that this can only fetch secrets from files that contain Kubernetes secrets, but it could be any file.

I've added a link to the existing docs about this topic. I'll open a PR to the docs repo before this is released.

Reply:
I didn't knew that we publicly documented the file aspect :+1: